newton/_src/solvers/kamino/_src/solvers/padmm/solver.py

```
@@ -242,7 +242,12 @@ def finalize(
         # Cache high-level solver options shared across all worlds
         self._warmstart = warmstart
         self._use_acceleration = use_acceleration
-        self._use_graph_conditionals = use_graph_conditionals
+        self._use_graph_conditionals = use_graph_conditionals and wp.is_conditional_graph_supported()
+        if use_graph_conditionals and not self._use_graph_conditionals:
+            msg.warning(
```

Ruff (0.16.1)

[error] Undefined name `msg`

(F821)